src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--31">
```

Comment:
please add start day modifier for calendar instead of add it for calendar__day
![image](https://user-images.githubusercontent.com/54936044/229274730-57aca6ee-7511-4059-b618-34017852b6ce.png)

Reply:
not fixed. instead modifiers in each calendar__day should be modifier in calendar.
if you have any questions, please ask in fe_chat